packages/metro-transformer/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@lingui/metro-transformer",
+  "version": "0.1.0",
```

Comment:
Please align with the current Lingui version and add publish config:

```
"publishConfig": {
  "access": "public"
},
```

This allows the package to be published by CI for the first time